packages/cli/src/utils/runCommandSync.ts

```diff
@@ -0,0 +1,74 @@
+import chalk from 'chalk'
+import type { ChildProcess, ExecException } from 'child_process'
+import { execSync } from 'child_process'
+
+type ExecSyncError = (ExecException & ChildProcess) | undefined
+
+const showError = ({
+  message,
+  cmd,
+  error,
+  debug,
+}: {
+  message: string
+  cmd: string
+  error: ExecSyncError
+  debug: boolean
+}) => {
+  console.log(`${chalk.red('error')} - ${message}`)
+
+  if (debug && cmd && error) {
+    console.log(`${chalk.magenta('DEBUG')} - $ ${cmd} error root ↓`)
```

Comment:
Not related to this PR, but we should start defining which colors we will use for which kind of information, right? Maybe have a higher abstraction?

Reply:
I think that would be a good idea.